src/config/project.rs

```diff
@@ -130,7 +130,7 @@ impl Project {
                 watch_additional_files,
                 hash_file,
                 hash_files: config.hash_files,
-                js_minify: cli.release && cli.js_minify && config.js_minify,
+                js_minify: cli.release && (cli.js_minify || config.js_minify),
```

Comment:
Ended up doing something slightly different than I originally proposed. [According to this comment in `cli.rs`](https://github.com/leptos-rs/cargo-leptos/blob/4b0c234bfd4efba05fbbfde2dbd625ab537c79e7/src/config/cli.rs#L59), the CLI option also only works for release builds. Now, it only ever works in release, but will work if either the CLI option or config option are specified.